minio/minio
# Implement Docker healthcheck script in Go

## Description
Implement Docker healthcheck script in Go

## Motivation and Context
Go script makes it easy to read/maintain. Also updated the timeout
in Dockerfiles from 5s to default 30s.

Higher timeout makes sense as server may sometimes respond slowly
if under high load as reported in #6974

Fixes #6974

## Regression
No

## How Has This Been Tested?
Local Docker test
- Build local Docker image using `TAG=custom-minio make docker`
- Run the image and confirm healthcheck status using `docker ps`

Swarm testing
- Deploy a Swarm cluster on https://labs.play-with-docker.com
- Confirm Minio cluster is deployed with healthy status

## Types of changes


## Checklist:
